models/Asset.php

```diff
@@ -1842,40 +1847,6 @@ public function setParent($parent)
         return $this;
     }
 
-    public function __sleep()
-    {
-        $parentVars = parent::__sleep();
-        $blockedVars = ['scheduledTasks', 'hasChildren', 'versions', 'parent', 'stream'];
-
-        if ($this->isInDumpState()) {
-            // this is if we want to make a full dump of the asset (eg. for a new version), including children for recyclebin
-            $this->removeInheritedProperties();
-        } else {
-            // this is if we want to cache the asset
-            $blockedVars = array_merge($blockedVars, ['children', 'properties']);
-        }
-
-        return array_diff($parentVars, $blockedVars);
-    }
-
-    public function __wakeup()
```

Comment:
we cannot remove that one, because this is different to the one in the `AbstractElement`